src/system/sys_profiler.cc

```diff
@@ -87,8 +99,8 @@ double MPI_Functions::max(double* val, MPI_Comm comm) {
  */
 
 Timer::Timer(const CodePoint &cp, int parent)
-: start_time(0),
-  cumul_time(0),
+: start_time(TimerData::get_time()),
+  cumul_time(start_time - start_time),
```

Comment:
Rather make cumul_time just double, time in seconds. See also notes in TimerData.